client/js/modules.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+// vendor libraries
+const $ = require("jquery");
+
+// our libraries
+const utils = require("./utils");
+
+module.exports = {
+	clear,
+	collapse,
+	expand,
+	join
+};
+
+function clear() {
+	utils.clear();
+}
+
+function collapse() {
+	$(".chan.active .toggle-button.opened").click();
+}
+
+function expand() {
+	$(".chan.active .toggle-button:not(.opened)").click();
+}
+
+function join(channel) {
+	var chan = utils.findCurrentNetworkChan(channel);
+
+	if (chan.length) {
+		chan.click();
+		return true;
+	}
```

Comment:
Okay so _this_ is what this PR/commit is about :)
Whenever possible, try to separate refactors from actual changes in different commits, that makes it for a better history and easier review :)